src/backend/common/helpers/event_helper.py

```diff
@@ -233,3 +236,29 @@ def week_events(cls) -> List[Event]:
                     events.append(event)
 
         return cls.sorted_events(events)
+
+    @classmethod
+    def remapteams_awards(
+        cls, awards: List[Award], remap_teams: Dict[str, str]
```

Comment:
Question asked in Slack - can we type this `str, str` situation so it's like, `old team, new team` or something?